tests/tester.js

```diff
@@ -55,7 +55,8 @@ const skipSlow = [
   'static_Call1MB1024Calldepth', // slow
   'static_Callcode50000', // slow
   'static_Return50000', // slow
-  'static_Return50000_2' // slow
+  'static_Return50000_2', // slow
+  'QuadraticComplexitySolidity_CallDataCopy'
```

Comment:
Why was this removed?

Reply:
This test is extremely slow, travis is getting on its limits (sometimes).